### PR TITLE
Translation fixes

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This PR simplifies some of the localizable strings so they are easier to translate.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/form/user-settings.js [hunk 1]
```diff
@@ -38,14 +38,35 @@ function formatElapsedTime(timestamp, intl) {
...
+  // This text can be shown in a tooltip as well as within HTML tags, therefore
   // it is obtained using formatMessage rather than <FormattedfMessage>.
-  return intl
-    .formatMessage(
-      { id: 'common.time.fromNowUpdate' },
-      { days, hours, minutes }
+  let duration
+  if (days === 0) {
+    if (hours === 0) {
+      if (minutes === 0) {
```
Comment (miles-grant-ibigroup) [inline lib/components/form/user-settings.js:46]: Real nitpick: couldn't we avoid all the else blocks if we put the conditions currently in the else blocks above the statements currently right after the if blocks?

File: lib/components/user/monitored-trip/saved-trip-list.js [hunk 2]
```diff
@@ -86,7 +86,13 @@ class TripListItem extends Component {
           <Panel.Title>
             <TripHeader>{trip.tripName}</TripHeader>
             <small>
-              From <b>{from.name}</b> to <b>{to.name}</b>
+              <FormattedMessage
+                id="components.SavedTripList.fromTo"
+                values={{
+                  from: <strong>{from.name}</strong>,
```
Comment (miles-grant-ibigroup) [inline lib/components/user/monitored-trip/saved-trip-list.js:92]: Is this allowed? Didn't we have an intl-specific strong tag?

**Review by miles-grant-ibigroup — COMMENTED** [review]
Tested and working, just a few nitpicks!

File: lib/components/user/monitored-trip/saved-trip-list.js (hunk 2 above)
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/saved-trip-list.js:92]: 100% allowed, and it simplifies the string resource.

File: lib/components/form/user-settings.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/form/user-settings.js]: Good suggestion, it's much cleaner that way (https://github.com/opentripplanner/otp-react-redux/pull/851/commits/2eb8d65740d2e3bf346c914811392329d87db26a).

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks great! Thanks for the changes

File: i18n/en-US.yml [hunk 3]
```diff
@@ -702,11 +695,8 @@ components:
   UserTripSettings:
     forgetOptions: Forget my options
     rememberOptions: Remember trip options
-    restoreOptions: |
-      Restore {defaults, select,
-        true {my defaults}
-        other {defaults}
-      }
+    restoreDefaultOptions: Restore defaults
+    restoreMyOptions: Restore my defaults
```
Comment (AdrianaCeric) [inline i18n/en-US.yml:699]:
```suggestion
    restoreMyDefaults: Restore my defaults
```

**Review by AdrianaCeric — APPROVED** [review]
Looks good! Just some naming

File: i18n/en-US.yml [hunk 4]
```diff
@@ -702,11 +695,8 @@ components:
   UserTripSettings:
     forgetOptions: Forget my options
     rememberOptions: Remember trip options
-    restoreOptions: |
-      Restore {defaults, select,
-        true {my defaults}
-        other {defaults}
-      }
+    restoreDefaultOptions: Restore defaults
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply i18n/en-US.yml]: Good suggestion, fixed in https://github.com/opentripplanner/otp-react-redux/pull/851/commits/4c9fe8de129c37aac3d5f85a415928c4efd53a36.

**Pull request merged**